pkgs/tools/bluetooth/blueman/default.nix

```diff
@@ -31,10 +31,11 @@ in stdenv.mkDerivation rec {
 
   configureFlags = [ (lib.enableFeature withPulseAudio "pulseaudio") ];
 
-  preFixup = ''
+  # This mimics ../../../development/interpreters/python/wrap.sh
+  postFixup = ''
     makeWrapperArgs="--prefix PATH ':' ${binPath}"
```

Comment:
@abbradar ah I forgot about that. Awhile ago I restricted `wrapPythonPrograms` to look only in `$out/bin` instead of the whole `$out`. Maybe it would make sense to revert that or indeed explicitly include `$out/libexec`.